Unit test `TestLinkProtection::test_handle_link_up_case_2` eventually fail

THe unit test `tests/unit/models/test_link_protection.py::TestLinkProtection::test_handle_link_up_case_2` eventually fails with the following output:

```
============================= test session starts ==============================
119platform linux -- Python 3.6.15, pytest-5.4.1, py-1.11.0, pluggy-0.13.1
120cachedir: .tox/py36/.pytest_cache
121rootdir: /builds/kytos/mef_eline/mef_eline, inifile: setup.cfg
122collected 118 items
123tests/unit/test_main.py ............................................     [ 37%]
124tests/unit/test_scheduler.py .........                                   [ 44%]
125tests/unit/test_storehouse.py ........                                   [ 51%]
126tests/unit/test_utils.py .                                               [ 52%]
127tests/unit/models/test_evc_base.py ............                          [ 62%]
128tests/unit/models/test_evc_deploy.py ..................                  [ 77%]
129tests/unit/models/test_link_protection.py ........F....                  [ 88%]
130tests/unit/models/test_path.py .............                             [100%]
131=================================== FAILURES ===================================
132________________ TestLinkProtection.test_handle_link_up_case_2 _________________
133self = <mef_eline.tests.unit.models.test_link_protection.TestLinkProtection testMethod=test_handle_link_up_case_2>
134deploy_to_path_mocked = <MagicMock name='deploy_to_path' id='140078278863616'>
135deploy_mocked = <MagicMock name='deploy' id='140078278864848'>
136    @patch('napps.kytos.mef_eline.models.evc.EVCDeploy.deploy')
137    @patch('napps.kytos.mef_eline.models.evc.EVCDeploy.deploy_to_path')
138    @patch('napps.kytos.mef_eline.models.path.Path.status', EntityStatus.UP)
139    def test_handle_link_up_case_2(self, deploy_to_path_mocked, deploy_mocked):
140        """Test if it is changing from backup_path to primary_path."""
141        deploy_mocked.return_value = True
142        deploy_to_path_mocked.return_value = True
143        primary_path = [
144                get_link_mocked(endpoint_a_port=9, endpoint_b_port=10,
145                                metadata={"s_vlan": 5},
146                                status=EntityStatus.UP),
147                get_link_mocked(endpoint_a_port=11, endpoint_b_port=12,
148                                metadata={"s_vlan": 6},
149                                status=EntityStatus.UP),
150        ]
151        backup_path = [
152                get_link_mocked(endpoint_a_port=9, endpoint_b_port=10,
153                                metadata={"s_vlan": 5},
154                                status=EntityStatus.UP),
155                get_link_mocked(endpoint_a_port=11, endpoint_b_port=12,
156                                metadata={"s_vlan": 6},
157                                status=EntityStatus.UP),
158        ]
159        attributes = {
160            "controller": get_controller_mock(),
161            "name": "circuit_name",
162            "uni_a": get_uni_mocked(is_valid=True),
163            "uni_z": get_uni_mocked(is_valid=True),
164            "primary_path": primary_path,
165            "backup_path": backup_path,
166            "enabled": True,
167            "dynamic_backup_path": True
168        }
169    
170        evc = EVC(**attributes)
171        evc.current_path = evc.backup_path
172        current_handle_link_up = evc.handle_link_up(primary_path[0])
173        self.assertEqual(deploy_mocked.call_count, 0)
174>       self.assertEqual(deploy_to_path_mocked.call_count, 1)
175E       AssertionError: 2 != 1
176tests/unit/models/test_link_protection.py:438: AssertionError
177=============================== warnings summary ===============================
178
```

--- a/tests/unit/models/test_link_protection.py
+++ b/tests/unit/models/test_link_protection.py
@@ -232,6 +232,7 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
 
         evc.current_path = evc.primary_path
         evc.activate()
+        deploy_to_mocked.reset_mock()
         current_handle_link_down = evc.handle_link_down()
         self.assertEqual(deploy_mocked.call_count, 0)
         deploy_to_mocked.assert_called_once()
@@ -267,13 +268,13 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
         ]
         backup_path = [
             get_link_mocked(
-                endpoint_a_port=9,
+                endpoint_a_port=7,
                 endpoint_b_port=10,
                 metadata={"s_vlan": 5},
                 status=EntityStatus.DOWN,
             ),
             get_link_mocked(
-                endpoint_a_port=11,
+                endpoint_a_port=15,
                 endpoint_b_port=12,
                 metadata={"s_vlan": 6},
                 status=EntityStatus.UP,
@@ -291,6 +292,7 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
 
         evc = EVC(**attributes)
         evc.current_path = evc.backup_path
+        deploy_to_mocked.reset_mock()
         current_handle_link_down = evc.handle_link_down()
         self.assertEqual(deploy_mocked.call_count, 0)
         deploy_to_mocked.assert_called_once()
@@ -349,6 +351,7 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
 
         evc = EVC(**attributes)
         evc.current_path = evc.backup_path
+        deploy_to_mocked.reset_mock()
         current_handle_link_down = evc.handle_link_down()
 
         self.assertEqual(get_paths_mocked.call_count, 0)
@@ -420,6 +423,7 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
         evc._storehouse.box = Mock()  # pylint: disable=protected-access
         evc._storehouse.box.data = {}  # pylint: disable=protected-access
 
+        deploy_to_mocked.reset_mock()
         current_handle_link_down = evc.handle_link_down()
         self.assertEqual(deploy_to_mocked.call_count, 1)
 
@@ -450,12 +454,12 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
         backup_path = [
             get_link_mocked(
                 endpoint_a_port=9,
-                endpoint_b_port=10,
+                endpoint_b_port=14,
                 metadata={"s_vlan": 5},
                 status=EntityStatus.UP,
             ),
             get_link_mocked(
-                endpoint_a_port=11,
+                endpoint_a_port=15,
                 endpoint_b_port=12,
                 metadata={"s_vlan": 6},
                 status=EntityStatus.UP,
@@ -474,6 +478,7 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
 
         evc = EVC(**attributes)
         evc.current_path = evc.primary_path
+        deploy_to_mocked.reset_mock()
         current_handle_link_up = evc.handle_link_up(backup_path[0])
         self.assertEqual(deploy_mocked.call_count, 0)
         self.assertEqual(deploy_to_mocked.call_count, 0)
@@ -503,12 +508,12 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
         backup_path = [
             get_link_mocked(
                 endpoint_a_port=9,
-                endpoint_b_port=10,
+                endpoint_b_port=14,
                 metadata={"s_vlan": 5},
                 status=EntityStatus.UP,
             ),
             get_link_mocked(
-                endpoint_a_port=11,
+                endpoint_a_port=15,
                 endpoint_b_port=12,
                 metadata={"s_vlan": 6},
                 status=EntityStatus.UP,
@@ -527,6 +532,7 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
 
         evc = EVC(**attributes)
         evc.current_path = evc.backup_path
+        deploy_to_path_mocked.reset_mock()
         current_handle_link_up = evc.handle_link_up(primary_path[0])
         self.assertEqual(deploy_mocked.call_count, 0)
         self.assertEqual(deploy_to_path_mocked.call_count, 1)
@@ -566,13 +572,13 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
         ]
         backup_path = [
             get_link_mocked(
-                endpoint_a_port=13,
+                endpoint_a_port=9,
                 endpoint_b_port=14,
                 metadata={"s_vlan": 5},
                 status=EntityStatus.DOWN,
             ),
             get_link_mocked(
-                endpoint_a_port=11,
+                endpoint_a_port=15,
                 endpoint_b_port=12,
                 metadata={"s_vlan": 6},
                 status=EntityStatus.UP,
@@ -596,6 +602,7 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
         evc._storehouse.box.data = {}  # pylint: disable=protected-access
 
         evc.current_path = Path([])
+        deploy_to_path_mocked.reset_mock()
         current_handle_link_up = evc.handle_link_up(backup_path[0])
 
         self.assertEqual(get_best_path_mocked.call_count, 0)
@@ -672,6 +679,7 @@ class TestLinkProtection(TestCase):  # pylint: disable=too-many-public-methods
         evc._storehouse.box = Mock()  # pylint: disable=protected-access
         evc._storehouse.box.data = {}  # pylint: disable=protected-access
 
+        deploy_to_path_mocked.reset_mock()
         current_handle_link_up = evc.handle_link_up(backup_path[0])
 
         self.assertEqual(get_best_path_mocked.call_count, 0)

